contracts/hydro/src/state.rs

```diff
@@ -48,10 +49,16 @@ pub struct Proposal {
     pub description: String,
     pub power: Uint128,
     pub percentage: Uint128,
+    pub bid_duration: u64, // number of rounds liquidity is allocated excluding voting round. default 1
```

Comment:
```suggestion
    pub bid_duration: u64, // number of rounds liquidity is allocated excluding voting round
```

the default is not actually enforced anywhere, right? I think it probably shouldn't mention the default here then